README.md

```diff
@@ -17,7 +17,7 @@ Clone pr-train's repo, and install it with `npm`.
 
 ```sh
 $ git clone git@github.com:Charitize/pr-train.git
-$ npm i -g pr-train
+$ npm i -g Charitize/pr-train
```

Comment:
This actually should be the relative path I think:

```suggestion
$ npm i -g ./pr-train
```

Reply:
Right, it would be better to run `cd pr-train` before. I'll change it to have both installation options.

Reply:
https://github.com/Charitize/pr-train/pull/13/files